python/wakapi_anyide/watchers/filewatcher.py

```diff
@@ -116,6 +120,9 @@ async def _task(self, queue: Queue[Event]):
                                 
                             self.current_file = resolved_path
                             self.current_file_bytes = await file.read()
+                        
+                        # Update the file in the cache
+                        self.cache[resolved_path] = await file.read()
```

Comment:
This is incorrect. The cache should be updated after an event is created for the file (and apparently I'd forgotten to do that, whoops)